src/models/questionsubmission/BuiltInFieldType.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Possible built-in field types for {@link DirectAnswer.fieldType}.
+ * @public
+ */
+export enum BuiltInFieldType {
```

Comment:
why is this added to questionsubmission?

Reply:
I think I accidentally dragged it in there for the last commit.